dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ProjectService.java

```diff
@@ -143,7 +143,7 @@
     public boolean hasProjectAndPerm(User loginUser, Project project, Map<String, Object> result) {
         boolean checkResult = false;
         if (project == null) {
-            putMsg(result, Status.PROJECT_NOT_FOUNT, project.getName());
+            putMsg(result, Status.PROJECT_NOT_FOUNT, "");
```

Comment:
We should tell users which projectName is not found ,not empty

Reply:
This method is called in many places. It may be query by projectId or the query by projectName, so it is difficult to pass the right project name.

Reply:
ok.